src/announcements/types.ts

```diff
@@ -0,0 +1,23 @@
+import {LanguageAndTextType} from '@atb-as/config-specs';
+import {FirebaseFirestoreTypes} from '@react-native-firebase/firestore';
+
+export type AnnouncementRaw = {
```

Comment:
I'm considering rewriting this as zod objects with transforms, rather than doing the mapping in `converters`.